src/tools/run-make-support/src/external_deps/rustc.rs

```diff
@@ -365,6 +365,9 @@ impl Rustc {
             if is_msvc() { None } else { Some("-lstatic:-bundle=stdc++") }
         } else if is_darwin() {
             Some("-lc++")
+        } else if is_aix() {
+            self.cmd.arg("-lc++");
```

Comment:
nit: this feels a little weird, because it doesn't really follow flow of the code. It would make more sense to me to return Some array of values and handle that below?

Reply:
Yes if there's more than one flag needed for AIX, just convert `flag` -> `flags` and construct a `vec![]` or something and change the `flag` match below.